frontend/src/learning/components/ItemDisplay/ProgressBar.js

```diff
@@ -0,0 +1,138 @@
+import React, {Component} from 'react'
```

Comment:
Please remember the trailing semis, and indention using 4 spaces :)

Reply:
4 space indention is crucial, and you should set it up in your IDE automatically